bookkeeper-server/src/test/java/org/apache/bookkeeper/bookie/EntryLogTest.java

```diff
@@ -391,4 +489,57 @@ public void testGetEntryLogsSet() throws Exception {
 
         assertEquals(Sets.newHashSet(0L, 1L, 2L, 3L), logger.getEntryLogsSet());
     }
+
+    /**
+     * Fake Ticker to advance ticker as expected.
+     */
+    class FakeTicker extends Ticker {
```

Comment:
I'm surprised gauva itself doesn't supply this, since testing is its reason for being.

Reply:
There exists indeed. Now I import the guava test-lib dependency to solve this.